site/assets/js/application.js

```diff
@@ -28,13 +28,22 @@
       new bootstrap.Popover(popover)
     })
 
+  var toastPlacement = document.getElementById('toastPlacement')
+  if (toastPlacement) {
+    document.getElementById('selectToastPlacement').addEventListener('change', function () {
+      if (!toastPlacement.dataset.originalClass) {
+        toastPlacement.dataset.originalClass = toastPlacement.className
+      }
+
+      toastPlacement.className = toastPlacement.dataset.originalClass + ' ' + this.value
```

Comment:
I assume you want to overwrite any previous classes here? If not, we could just use `classList.add()`?

Reply:
Yep, the class needs to be overridden since position utilities can be added.